src/components/transactions/messageTypes.js

```diff
@@ -15,21 +15,4 @@ const messageType = {
   WITHDRAW_VALIDATOR_COMMISSION: "cosmos-sdk/MsgWithdrawValidatorCommission"
 }
 
-const transactionGroup = {
```

Comment:
This is still imported in `src/scripts/transaction-utils.js`. You may need to delete it there also.